src/components/molecules/ChatPoll/ChatPoll.scss

```diff
@@ -0,0 +1,113 @@
+@import "scss/constants";
+
+$poll-text-border-radius-me: $spacing--xxl $spacing--xxl $spacing--xs
+  $spacing--xxl;
+$poll-text-border-radius: $spacing--xxl $spacing--xxl $spacing--xxl $spacing--xs;
+
+button.ChatPoll__question {
+  display: block;
+  margin-bottom: $spacing--lg;
+  width: 100%;
+  background-color: $secondary--light;
+
+  &:hover {
+    background-color: $secondary--lightest;
+  }
+}
```

Comment:
Why do you need to have it like this? Why not just `__question` inside the `ChatPoll`?

Reply:
to override the global styles button background

Reply:
Please add the appropriate comment there in this case

Reply:
fixed in 5c01bebb151762551ca7c3128fa6813b81a99672